src/views/product-downloads-view/index.tsx

```diff
@@ -89,7 +89,7 @@ const ProductDownloadsViewContent = ({
 				versionSwitcherOptions={versionSwitcherOptions}
 			/>
 			<OfficialReleasesSection />
-			{featuredLearnCards ? (
+			{featuredLearnCards && featuredLearnCards.length > 0 ? (
```

Comment:
I think this can be shortened if you want.

```tsx
			{featuredLearnCards?.length > 0 ? (
```

Reply:
```suggestion
			{featuredLearnCards?.length > 0 ? (
```